src/test/kotlin/chapter3/solutions/Solution_3_13.kt

```diff
@@ -10,6 +10,9 @@ fun <A> append(a1: List<A>, a2: List<A>): List<A> =
     foldRight(a1, a2, { x, y -> Cons(x, y) })
 // end::init[]
 
+// Solution using foldLeft():
+//fun <A> append(a1: List<A>, a2: List<A>): List<A> = foldLeft(reverse(a1), a2, {y, x -> Cons(x, y)})
```

Comment:
I feel that it is important that all code snippets compile. I would uncomment this line and simply give it some other name like `appendFoldLeft`. wdyt?

Reply:
Sure, I'll have a look at it tonight. I'll then commit the change to the branch, then everything should be ready for merging into the master branch.